packages/core/admin/admin/src/components/GuidedTour/Homepage.tsx

```diff
@@ -5,20 +5,34 @@ import { ArrowRight } from '@strapi/icons';
 import { useIntl } from 'react-intl';
 import { NavLink } from 'react-router-dom';
 
-import { LAYOUT_DATA, States, STATES } from './constants';
+import { SUPER_ADMIN_LAYOUT_DATA, LAYOUT_DATA, States, STATES } from './constants';
 import { Number, VerticalDivider } from './Ornaments';
 
-const GuidedTourHomepage = () => {
-  const { guidedTourState, setSkipped } = useGuidedTour();
+interface GuidedTourHomepageProps {
+  visibility: string;
+}
+
+const GuidedTourHomepage = ({ visibility }: GuidedTourHomepageProps) => {
+  const { guidedTourState, setSkipped, guidedTourVisibility } = useGuidedTour();
   const { formatMessage } = useIntl();
   const { trackUsage } = useTracking();
 
-  const sections = Object.entries(LAYOUT_DATA).map(([key, val]) => ({
+  const layout = visibility === 'super-admin' ? SUPER_ADMIN_LAYOUT_DATA : LAYOUT_DATA;
+  const layoutCopy = JSON.parse(JSON.stringify(layout));
+
+  const triggeredBySA = guidedTourVisibility === 'super-admin' ? true : false;
+
+  // Remove the inviteUser step  if we are in the development env
+  if (process.env.NODE_ENV === 'development') {
+    delete layoutCopy.inviteUser;
+  }
```

Comment:
This isn't a runtime check so it won't work how you expect it too

Reply:
Is there a way to make it work like I expect it to be?

Reply:
you'd need to check the runtime version i think you can use `useAppInfo()` and access `currentEnvironment`, iirc.